notebook/secciones/geometria/poly.cpp

```diff
@@ -67,6 +67,34 @@ struct poly{
 		}
 		return poly(ret);
 	}
+	ii cut(pto u, pto v) { // O(log(n)) for convex polygons
```

Comment:
Agregale una descripcion de lo que retorna porfa, algo tipo:
```suggestion
        // returns range [from, to] such that pt[from], ..., pt[to] are all
        // the points to the left of line u->v (note that it could happen
        // that from > to as indexes are circular in a polygon)
	ii cut(pto u, pto v) { // O(log(n)) for convex polygons
```